src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -0,0 +1,257 @@
+﻿using Dynamo.Models;
+using Dynamo.ViewModels;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Dynamo.Core
+{
+    internal class CrashReportArgs : EventArgs
+    {
+        public bool SendLogFile = true;
+        public bool SendSettingsFile = true;
+        public bool SendDynFile = true;
+        public bool SendRecordedCommands = true;
+
+        internal DynamoViewModel viewModel;
+
+        internal CrashReportArgs(DynamoViewModel dynamoViewModel)
+        {
+            viewModel = dynamoViewModel;
+        }
+    }
+
+    internal class CrashReportTool
+    {
+        static CrashReportTool()
+        {
+            try
+            {
+                CerToolDir = GetCERToolLocation();
+            }
+            catch
+            {}
+        }
+
+        private static List<string> ProductsWithCER => new List<string>() { "Revit", "Civil", "Robot Structural Analysis" };
```

Comment:
hmm would this ever be different from the ASM product list?

Reply:
I think more adsk products might have CER than ASM.
A counter example is Formit, which only started to integrate CER this quarter.
I am hoping that this code (that searches for CER) will be removed once we get approved for CER integration in Dynamo as open source binaries)